app/models/cv_validation_job.rb

```diff
@@ -50,21 +54,24 @@ class CvValidationJob
   #   @return [Array<String>] The messages that were logged during the CV Validation Job. This is a placeholder for future enhancements/use.
   field :logging_messages, type: Array
 
-  # @!attribute [rw] cv_start_time
-  #   @return [DateTime] The start timestamp to track how long the CV payload took to generate.
-  field :cv_start_time, type: DateTime
+  # @!attribute [rw] cv_payload_transformation_time
```

Comment:
I'm glad we have opted to store elapsed time instead of the start/end time stamps.  It will remove the additional overhead of calculating the elapsed time after the fact.